spec/classroom_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../classroom'
+
+describe Classroom do
+  before :each do
```

Comment:
Please add test for `add_student` method of `Classroom` class.

Reply:
Thank you for letting us know. We did the test for the add_student method of the Classroom class!